UI/window-basic-properties.cpp

```diff
@@ -114,15 +114,20 @@ OBSBasicProperties::OBSBasicProperties(QWidget *parent, OBSSource source_)
 		obs_display_add_draw_callback(preview->GetDisplay(),
 				OBSBasicProperties::DrawPreview, this);
 	};
-
+	
 	enum obs_source_type type = obs_source_get_type(source);
 	uint32_t caps = obs_source_get_output_flags(source);
 	bool drawable_type = type == OBS_SOURCE_TYPE_INPUT ||
 		type == OBS_SOURCE_TYPE_SCENE;
-
-	if (drawable_type && (caps & OBS_SOURCE_VIDEO) != 0)
+	bool drawablePreview = (caps & OBS_SOURCE_VIDEO) != 0;
+	
+	if (drawablePreview && drawable_type) {
+		preview->show();
 		connect(preview.data(), &OBSQTDisplay::DisplayCreated,
-				addDrawCallback);
+			addDrawCallback);
```

Comment:
The indentation of the connect call didn't change, so this line shouldn't change either; please remove this change from the commit